src/components/Auth/AuthHookForm.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+import { LinkbraryIcon } from "@/src/asset";
+import Link from "next/link";
+import { useForm } from "react-hook-form";
+import InputField from "./InputField/InputField";
+import styles from "./AuthHookForm.module.scss";
+
+export default function AuthHookForm() {
+  const { handleSubmit } = useForm();
+  // register : value 관련
+  // watch : input 상태감지
+  // formState : submit error 감지
+
+  return (
+    <main className={styles.authForm}>
+      <header className={styles.header}>
+        <Link href={"/"}>
+          <LinkbraryIcon width="212" height="38" />
+        </Link>
+        <div className={styles.header__text}>
+          <p>회원이 아니신가요?</p>
+          <Link href={"/signup"}>회원 가입하기</Link>
```

Comment:
문자열은 중괄호없이 표현해주고 상수화해서 다뤄주면 더 좋을 것 같습니다~
```suggestion
          <Link href="/signup">회원 가입하기</Link>
```